### DISCUSSION TITLE
Is there any way to sample along a normal distribution from a built-in mitsuba/drjit function?

### DISCUSSION THREAD

**Discussion by Mephisto405** [mitsuba-renderer/mitsuba3#701] [Q&A] [answered]
I want to add Gaussian noise to a Vector3f or Spectrum.
I can make a normal distribution warping function from a uniform sample as below, but I want to use a built-in function if it exists (and more optimal).
```python
dr.sqrt(2)*dr.erfinv(dr.fma(2, sampler.next_1d(), -1))
```
I'd searched the official docs of Mitsuba and drjit, but I cannot find one.

**Comment by njroussel** [comment, marked as answer]
Hi @Mephisto405 

We have utilities to build distributions: https://mitsuba.readthedocs.io/en/stable/src/api_reference.html#distributions.
But I don't think we provide any built-in distributions.

**Reply by Mephisto405** [reply]
Oh kkk I see. Thanks for confirming that!!
